metrics/influxdb_client.go

```diff
@@ -0,0 +1,93 @@
+package metrics
+
+import (
+	"os"
+	"time"
+
+	influxdb "github.com/influxdata/influxdb-client-go/v2"
+)
+
+const EnvInfluxDBURL = "INFLUXDB_URL"
+const EnvInfluxDBToken = "INFLUXDB_TOKEN"
+const EnvInfluxDBOrg = "INFLUXDB_ORG"
+const EnvInfluxDBBucket = "INFLUXDB_BUCKET"
+
+
+type Config struct {
+  InfluxURL string
+  InfluxToken string
+  Org string
+  Bucket string
+}
+
+type MetricsRecorder interface {
+  Record(string, map[string]string, map[string]interface{})
+  URL() string
```

Comment:
Not a fan of this method cause it seems kind of random on a metrics recorder interface but if you absolutely want it sure

Reply:
def want it atm

Reply:
just want it for logging purposes atm